src/util/saveArticles.ts

```diff
@@ -562,52 +606,21 @@ export function applyOtherTreatments(parsoidDoc: DominoElement, dump: Dump) {
   /* Remove empty paragraphs */
   // TODO: Refactor this option to work with page/html and page/mobile-html output. See issues/1866
   if (!dump.opts.keepEmptyParagraphs) {
-    if (!dump.opts.keepEmptyParagraphs) {
-      // Mobile view === details
-      // Desktop view === section
-      const sections: DominoElement[] = Array.from(parsoidDoc.querySelectorAll('details, section'))
-      for (const section of sections) {
-        if (
-          section.children.length ===
-          Array.from(section.children).filter((child: DominoElement) => {
-            return child.matches('summary')
-          }).length
-        ) {
-          DU.deleteNode(section)
-        }
+    // Mobile view === details
```

Comment:
Should we not move this (remove empty sections) in the renders code?

Reply:
Yes, this moved to the abstract renderer now after rebasing on top of recent changes with https://github.com/openzim/mwoffliner/pull/1886